app/styles/wheel-animations.css

```diff
@@ -0,0 +1,79 @@
+.shine {
+  background: #f6f7f8;
+  background-image: linear-gradient(
+    to right,
+    #f6f7f8 0%,
+    #edeef1 20%,
+    #f6f7f8 40%,
+    #f6f7f8 100%
+  );
+  background-repeat: no-repeat;
+  background-size: 150px 150px;
```

Comment:
can we use rem?

Reply:
Should the size of the image change, when the default font size is changed